tests/sqllogictests/suites/mode/cluster/explain_v2.test

```diff
@@ -342,50 +339,40 @@ Limit
     ├── output columns: [numbers.number (#0), numbers.number (#1)]
     ├── sort keys: [number ASC NULLS LAST]
     ├── estimated rows: 50.00
-    └── Exchange
-        ├── output columns: [numbers.number (#0), numbers.number (#1), #_order_col]
-        ├── exchange type: Merge
-        └── Sort
-            ├── output columns: [numbers.number (#0), numbers.number (#1), #_order_col]
-            ├── sort keys: [number ASC NULLS LAST]
-            ├── estimated rows: 50.00
-            └── HashJoin
-                ├── output columns: [numbers.number (#0), numbers.number (#1)]
-                ├── join type: RIGHT OUTER
-                ├── build keys: [CAST(y.a (#1) AS UInt64 NULL)]
-                ├── probe keys: [x.a (#0)]
-                ├── filters: []
-                ├── estimated rows: 50.00
-                ├── Exchange(Build)
-                │   ├── output columns: [numbers.number (#1)]
-                │   ├── exchange type: Hash(y.a (#1))
-                │   └── TableScan
-                │       ├── table: default.system.numbers
-                │       ├── output columns: [number (#1)]
-                │       ├── read rows: 5
-                │       ├── read bytes: 40
-                │       ├── partitions total: 1
-                │       ├── partitions scanned: 1
-                │       ├── push downs: [filters: [], limit: NONE]
-                │       └── estimated rows: 5.00
-                └── Exchange(Probe)
-                    ├── output columns: [numbers.number (#0)]
-                    ├── exchange type: Hash(x.a (#0))
-                    └── TableScan
-                        ├── table: default.system.numbers
-                        ├── output columns: [number (#0)]
-                        ├── read rows: 10
-                        ├── read bytes: 80
-                        ├── partitions total: 1
-                        ├── partitions scanned: 1
-                        ├── push downs: [filters: [], limit: NONE]
-                        └── estimated rows: 10.00
+    └── HashJoin
```

Comment:
Is the test as expected? `Merge` too early to take advantage of the benefits of distribution

Reply:
It occurs when the table is very small and can not fit the cost model well. If we tune the data set to a larger scale, this will be fixed.

Reply:
In this case, the overhead of merge data after join is bigger than merge the data before join. I'll modify the test to make it more reasonable.